src/llmq/quorums_signing_shares.cpp

```diff
@@ -677,7 +677,7 @@ bool CSigSharesManager::ProcessPendingSigShares(CConnman& connman)
             if (!pubKeyShare.IsValid()) {
                 // this should really not happen (we already ensured we have the quorum vvec,
                 // so we should also be able to create all pubkey shares)
-                LogPrintf("CSigSharesManager::%s -- pubKeyShare is invalid, which should not be possible here");
+                LogPrintf("CSigSharesManager::%s -- pubKeyShare is invalid, which should not be possible here", __func__);
```

Comment:
It looks like a prior fix is missing here (`\n` at the end). Might also want to (partially) backport https://github.com/dashpay/dash/pull/3520/commits/ec7c7dfc8d18c22754a432e781192d0b31eadadd and https://github.com/dashpay/dash/pull/3520/commits/2beab95acb7e125fb9f122b11f25434af203c471 from #3520 to fix similar issues in other places.

Reply:
sure. Cherry-picked both